rstcheck/__init__.py

```diff
@@ -479,7 +488,7 @@ def find_config(  # noqa: CCR001
             break
         directory = parent_directory
 
-    return None
+    return ""
```

Comment:
I really don't like the idea of using an empty string instead of a `None` to signal that nothing was found.
And I do not see any hindrance to keep using `None` (see other comments).

Reply:
This is a must change for me.

Reply:
This is a must change for me.